lib/ruby-lint/definition/ruby_object.rb

```diff
@@ -677,6 +683,12 @@ def inspect
         return %Q(#<#{self.class}:0x#{address} #{attributes.join(' ')}>)
       end
 
+      def top_scope
+        return self if type == :root
+        scope = parents.last   # the enclosing scope?
```

Comment:
What's the reason for this comment? Right now it doesn't really convey any useful information since it's a question.